### ISSUE TITLE
HP values being unchanged

### ISSUE THREAD

**Issue by jamre45290** [Vectorial1024/EliteBionicsFramework#21]
midway through this playthrough i am doing with Science Never Stops, i have noticed that the bionics in it have the same value to it's original counterpart rather than the intended changed value this mod is meant to set

**Comment by jamre45290** [comment]
i have decided to sift through the errors and it appears there are a few mods with unmodified values, such as Immortals, Pawnmorpher (even though the steam page said it was pathed), even after disabling these mods the HP values of bionics still havent changed at all

**Comment by Vectorial1024** [comment]
Is related to #4 

The thing is, Science Never Stops is closed-source, and there seems no activity on their side, so I am not sure what is going on about this. There was one moment that I changed the code name of some stuff on my side, but then again there seemingly were no updates on their side, so again idk.

------

btw, when I check the Pawnmorpher Steam page just now, they did not mention having compatibility with Elite Bionics Framework, so technically Pawnmorpher compatibility is "working as intended". Besides, Pawnmorpher is currently not listed as one of the "known mods that usess EBF", so it is most likely working as intended.

https://steamcommunity.com/sharedfiles/filedetails/?id=1786466855

**Comment by jamre45290** [comment]
but still, even the bionics from RSBE have default values unless that mod is also not patched?

**Comment by jamre45290** [comment]
i managed to get in-contact with the dev of SNS, and they said EBF not working for SNS is likely from an update, SNS wont be able to create a patch for EBF compatibility on their end due to them working on a new major update which wont be anytime soon

**Comment by Vectorial1024** [comment]
The bottom line is, many many other bionic mods out there will work fine with this mod with the reason being "they did not notice this mod and did not use this mod". Those bionics will have their max HP values unchanged even when installed. If, one day, the bionic mod makers noticed this EBF mod and decided to add code on their side, then their mod will truly work fine with this mod and you will notice the bionics have increased max HP. This will have to be done by the other mod makers because it involves balancing.

-------

I separately checked the logs on my side (good that I kept some logs...) and yes, currently EBF should not throw any errors with Pawnmorpher, but I must admit, there were some guesswork involved with the compatibility patch: I had to assume that Pawnmorpher did not insert any extra meaning to "body part max HP"... What if they eg use body part max HP to determine if a body part is "pure", and then there will be some inaccuracy/incorrectness introduced by my patch.

---------

Aigh, it is understandable when SNS says they are working on something new, and cannot respond quickly. To be honest I have also kind of overextended myself on modding because I really did not expect real life to turn so suddenly... it is not just here, but my other modding projects (may include other games) all have issues piled up and I cannot respond to them quickly.

Hopefully the SNS team knows about this potential problem and checks it before they release their next major update.
